apps/protocol-frontend/src/components/VerificationFramework.tsx

```diff
@@ -35,7 +57,13 @@ const VerificationFramework = () => {
             option you can always return and edit settings.
           </Text>
         </Flex>
-        <VerificationFrameworkForm />
+        <VerificationFrameworkForm
+          verificationSettingId={verificationSettingId ?? null}
+          daoId={daoId}
+          numberOfAttestations={
+            verificationSettingData?.num_of_attestations ?? null
+          }
+        />
```

Comment:
This form seems to jump when switching frameworks. Happens towards the end of the video

[Screencast from 2023-03-27 12-51-40.webm](https://user-images.githubusercontent.com/12705146/228011036-1a1f85ab-014f-458a-bd20-49a2c3a6743b.webm)

Reply:
Ah, yup. I think that this is being caused by the conditional reveal of the second input. Can tweak this